projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.7-a.5 - 2022-02-16
+### Enhancements
+- Form block: add support for color and spacing block options.
+- Mailchimp block: add align, color and spacing features
+- Podcast Player block: add Align (Wide and Full Width) and Spacing (Padding and Margin) tools
```

Comment:
Since those (and the Repeat Visitor block below) are really the same thing, maybe we can group them, or give them the same description?